packages/atlas/src/components/Text/Text.styles.ts

```diff
@@ -46,72 +74,72 @@ const clampStyles = ({ clampAfterLine }: TextBaseProps) => css`
   overflow: hidden;
 `
 
-const baseStyles = ({ color, secondary, clampAfterLine, margin, align }: TextBaseProps) => css`
-  color: ${color ? color : secondary ? cVar('colorText') : cVar('colorTextStrong')};
+const baseStyles = ({ color = 'strong', clampAfterLine, margin, align }: TextBaseProps) => css`
```

Comment:
I know that changing this will require a lot of work, but It would be much more intuitive for me if the default value were indeed `default` not `strong`.

Reply:
As discussed on atlas dev sync, we decided to not implement this